src/components/CourseChecklist/index.jsx

```diff
@@ -327,6 +332,13 @@ class CourseChecklist extends React.Component {
     return false;
   }
 
+  shouldShowCheck = (checkID) => {
+    if (checkID === 'certificates') {
+      return this.props.studioDetails.links.certificates.trim() !== '';
```

Comment:
This condition will have to change based on the feedback Farhanah gave you on https://github.com/edx/edx-platform/pull/18842 in order to check for something like `this.props.data.certificates.is_enabled` instead.